md/bpm-api.md

```diff
@@ -3092,6 +3092,44 @@ You can search cases.
     ]
     ```
 
+#### Retrieve an archived case context
+
+* **URL**  
+  `/API/bpm/archivedCase/:archivedCaseId/context`  
+* **Method**  
+  `GET`
+* **Success Response**  
+  A context object
+  * **Code**: 200
+  * **Payload**:  
+    ```json
+    {  
+      "businessData_ref":{  
+        "name":"myBusinessDate",
+        "type":"com.company.model.BusinessObject1",
+        "link":"API/bdm/businessData/com.company.model.BusinessObject1/2",
+        "storageId":2,
+        "storageId_string":"2"
+      },
+      "document1_ref":{  
```

Comment:
Actually this is the response payload :-).  So i guess using payload here is right